### ISSUE TITLE
fix report links

### ISSUE THREAD

**Issue by alorenzo175** [SolarArbiter/solarforecastarbiter-core#175]
Currently, reports on the dev dashboard link forecast/observations to the prod dashboard

**Comment by wholmgren** [comment]
In `template.md`:

```
{# fix this #}
{% set dash_url = 'https://dashboard.solarforecastarbiter.org' %}
```

**Comment by wholmgren** [comment]
@alorenzo175 suggested relative links. This can easily be accomplished by setting `template.md` `dash_url = ''` and removing the leading `/` from the remainder of the urls. So, that will resolve issues around dashboard vs. dev-dashboard. The downside is that breaks links for locally generated reports using real API data (like what you get running the CLI reports option). For example, the link for [`University of Arizona OASIS ghi`](https://dashboard.solarforecastarbiter.org/observations/9f657636-7e49-11e9-b77f-0a580a8003e9) turns into `file:///Users/holmgren/git_repos/solarforecastarbiter-core/observations/9f657636-7e49-11e9-b77f-0a580a8003e9`

**Comment by alorenzo175** [comment]
closed in #321
